src/components/GenerateCVButton/container.js

```diff
@@ -21,10 +25,10 @@ const GenerateButtonContainer = Wrapped =>
     };
 
     handleOnClick = () => {
-      const { selectedEmployee, selectedTemplate } = this.props;
+      const { selectedEmployee, selectedTemplate, employeeCV } = this.props;
+
       // info that will eventually be sent to the backend in the POST
-      console.log('selected employee: ', selectedEmployee);
-      console.log('selectedTemplate: ', selectedTemplate);
+      employeeCV(selectedEmployee, selectedTemplate);
```

Comment:
What's this function doing? I can't really tell by the name of it.

Reply:
this function passes the selected items from the dropdowns as props and fires off the action creator in the mapDispatchtoProps. I have renamed it to `generateEmployeeCV` which hopefully explains it better